client/js/libs/handlebars/parse.js

```diff
@@ -81,5 +81,8 @@ module.exports = function parse(text) {
 		}
 
 		return fragments;
+	}).join("") + linkParts.map((part) => {
+		const escapedLink = Handlebars.Utils.escapeExpression(part.link);
+		return `<div data-url="${escapedLink}"></div>`;
```

Comment:
Actually, escaping this is going to make finding stuff with jQuery difficult (it's matching based on `preview.link`, which is unescaped). Do we actually need to escape this, here? It's not for show, it's mostly a key.